server/integration_tests/standalone/adversarial.py

```diff
@@ -434,26 +444,32 @@ def read_tsv(csv_file: str):
 
 
 def run_test():
-  os.makedirs(os.path.join(FLAGS.output_dir, REPORTS_DIR), exist_ok=True)
-  test = AdversarialQueriesTest(base_url=FLAGS.base_url, llm_api=FLAGS.llm_api)
+  output_dir = FLAGS.output_dir
+  if FLAGS.use_sdg:
+    output_dir += "_sdg"
```

Comment:
Can we do sub-directories instead of peer directories? i.e. `{output_dur}/{dc}` (`output/main`, `output/sdg`, etc.)? Again, will make it easier to support more DCs.

Reply:
Done